common/src/main/java/com/viaversion/viaversion/commands/defaultsubs/PlayerSubCmd.java

```diff
@@ -46,12 +46,19 @@ public boolean execute(final ViaCommandSender sender, final String[] args) {
         if (args.length == 0) {
             return false;
         }
+
+        final boolean all = args[0].equals("*");
+        boolean any = false;
         for (final UserConnection connection : Via.getManager().getConnectionManager().getConnections()) {
             final ProtocolInfo info = connection.getProtocolInfo();
-            if (args[0].equalsIgnoreCase(info.getUsername()) || args[0].equals("*")) {
-                sendMessage(sender, "&7[&6" + info.getUsername() + "&7] UUID: &2" + info.getUuid() + " &7Client-Protocol: &2" + info.protocolVersion().getName() + " &7Server-Protocol: &2" + info.serverProtocolVersion().getName());
+            if (args[0].equalsIgnoreCase(info.getUsername()) || all) {
+                sendMessage(sender, "&7[&6" + info.getUsername() + "&7] UUID: &2" + info.getUuid() + " &7Client-Protocol: &2" + info.protocolVersion().getName() + " &7Server-Protocol: &2" + info.serverProtocolVersion().getName() + " &7Clientside: &2" + connection.isClientSide());
```

Comment:
Shouldn't server/client protocol texts have no dashes in them? Also `Clientside` -> `Client-Side`.

Reply:
Should be fine now